sw.js

```diff
@@ -1,103 +1,47 @@
 importScripts(
-  'js/vendor/sw-cacheable-response.min.js',
-  'js/vendor/sw-routing.min.js',
-  'js/vendor/sw-runtime-caching.min.js',
-  'js/vendor/sw-precaching.min.js'
+  'https://storage.googleapis.com/workbox-cdn/releases/3.3.1/workbox-sw.js'
 );
 
-// Make this always match package.json version
-const version = '0.1.3';
-const cacheName = `defaultCache_${version}`;
```

Comment:
Oh, one more thing I noticed! I ran the site locally, and the `defaultCache_*` cache is sticking around. 🙈 

![screen shot 2018-07-12 at 3 44 33 pm](https://user-images.githubusercontent.com/459757/42663942-e99822b8-85ec-11e8-8edb-6f5107130a3e.png)

We'll want to purge the out-of-date caches as well. Workbox does not handle this for us. Instead, it is our responsibility to remove out-of-date caches. 

In another project, I set up a two-level version cache naming system. Here are some snippets in case it helps (the `module.exports` may or may not apply):

```js
// sw/caches.js

/**
 * This sets up a two-level version cache naming system.
 * Edit the GLOBAL_VERSION to purge all caches.
 * Edit the individual cache version to purge the specific cache.
 */
const GLOBAL_VERSION = 1;
const CACHES = {
  PRECACHE: `pwastats-precache-v${GLOBAL_VERSION + 0}`,
  FONTS: `pwastats-fonts-v${GLOBAL_VERSION + 1}`,
  STATIC_ASSETS: `pwastats-static-assets-v${GLOBAL_VERSION + 0}`,
  IMAGES: `pwastats-images-v${GLOBAL_VERSION + 0}`
};

/**
 * This allows the CACHES to be available both in the
 * workbox.config.js as well as within the service worker.
 */
try {
  module.exports = CACHES;
} catch (e) {
  // Do nothing
  // For when CACHES is used within service worker scope.
}
```


```js
// sw/sw-delete-caches.js

importScripts('/sw/caches.js');

const validCacheList = Object.values(CACHES);

/**
 * Checks the cache name in question against the valid list of caches
 * @param {String} cacheName The name of the cache in question
 * @returns {Boolean} Is the `cacheName` valid?
 */
const cacheIsValid = cacheName =>
  validCacheList.some(validCacheName =>
    cacheName.includes(validCacheName)
  );

/**
 * Delete out-of-date caches
 */
self.addEventListener('activate', event => {
  event.waitUntil(
    caches.keys().then(cacheNameList => Promise.all(
      cacheNameList.map(cacheName => {
        if (!cacheIsValid(cacheName)) {
          console.info(`SW :: Deleting cache "${cacheName}"`);
          return caches.delete(cacheName);
        }
      })
    ))
  );
});
```


```js
// sw.js
importScripts("https://storage.googleapis.com/workbox-cdn/releases/3.2.0/workbox-sw.js");
importScripts("/sw/sw-delete-caches.js");
```